src/pokete/classes/fight/fight.py

```
@@ -187,31 +190,45 @@ def __call__(self, ctx: Context, providers: list[Provider]):
+            success = False
+            if g["type"] == "BLITZ":
+                success = turns <= g["target"]
+            elif g["type"] == "TACTICIAN":
+                success = len(unique_moves) >= g["target"]
+            elif g["type"] == "SURVIVOR":
+                success = player_end_hp >= (player_start_hp * g["target"])
+
+            if won: # Only count success if battle won
+                dm.report_gambit_result(success)
+            else:
+                dm.report_gambit_result(False)
```

Ruff (0.14.14)

[error] Boolean positional value in function call

(FBT003)